api/src/main/java/com/talentpath/amazin/api/customerWhoBoughtAlso/CustomerWhoBoughtAlso.java

```diff
@@ -1,5 +1,4 @@
 package com.talentpath.amazin.api.customerWhoBoughtAlso;
-import java.util.ArrayList;
```

Comment:
I actually need this import for my app. its no big deal, but just as a heads up.

Reply:
Sure, that might have been a mistake on my end. Thank you!